src/portal/src/app/project/repository/artifact/artifact-additions/artifact-vulnerabilities/artifact-vulnerabilities.component.html

```diff
@@ -10,10 +10,10 @@
         <clr-datagrid [clrDgLoading]="loading">
             <clr-dg-action-bar>
                 <div class="clr-row center">
-                    <div class="clr-col-1">
+                    <div class="ml-05">
                         <button (click)="scanNow()" type="button"  class="btn  btn-secondary" [clrLoading]="scanBtnState" [disabled]="!(hasEnabledScanner && hasScanningPermission && !onSendingScanCommand)"><clr-icon shape="shield-check" size="16"></clr-icon>&nbsp;{{'VULNERABILITY.SCAN_NOW' | translate}}</button>
                     </div>
-                    <div class="clr-col">
+                    <div class="ml-1">
```

Comment:
ml-1 is not defined in artifact-vulnerabilities.component.scss

Reply:
For widely used css,  them are in global css